labonneboite/importer/settings.py

```diff
@@ -29,16 +31,24 @@
 MYSQL_NO_PASSWORD = False
 
 
-def get_departements():
-    departements_str = []
-    departements = range(1, 96)
-    departements.append(97)  # DOM-TOM
-    for d in departements:
+@lru_cache(maxsize=None)
+def get_departements(largest_ones_first=False):
+    departements = []
+    raw_departements = range(1, 96) + [97]  # 97 is DOM-TOM
+    for d in raw_departements:
         if d < 10:
             dep = "0%s" % d
         else:
             dep = str(d)
-        departements_str.append(dep)
-    return departements_str
+        departements.append(dep)
```

Comment:
Tout ce bout de code peut être remplacé par : 

    departements = ["{:02d}".format(d) for d in range(1, 96)] + ['97']

Reply:
will do